<?xml version="1.0"?>

<launch>
    <node name="path_setter" pkg="icuas24_competition" type="path_setter.py" args="--use-points" output="screen"/>

    <node name="photo_taker" pkg="icuas24_competition" type="photo_taker.py" output="screen"/> 

    <node name="image_analyzer" pkg="icuas24_competition" type="image_analyzer.py" output="screen" />

    <node name="photo_logger" pkg="icuas24_competition" type="photo_logger.py" output="screen" />
</launch>
